pkg/chartmuseum/router/match.go

```diff
@@ -125,6 +125,9 @@ func match(routes []*Route, method string, url string, contextPath string, depth
 		if route.Method != method {
 			continue
 		}
+		if checkStaticRoute(route.Path) {
```

Comment:
Is it possible to update the unit tests to cover this case?

Reply:
thanks for pointing out, ofc will do!

Reply:
done